### ISSUE TITLE
Not working with portals

### ISSUE THREAD

**Issue by cluk3** [rokborf/natural-drag-animation-rbdnd#6]
Hey, first of all thanks a lot for the great package, it's super useful!
In my app I'm rendering the draggable into a portal as described [here](https://github.com/atlassian/react-beautiful-dnd/blob/master/docs/guides/reparenting.md), but doing so will cause the transform property to be always null, therefore the draggable will be stuck in is original position.
I have done some debugging and looks like the problem is that when rendering the draggable in a portal, it is mounted with `snapshot.isDragging` already set to true, which will cause the animationFrame callback to never initiate: 

https://github.com/rokborf/natural-drag-animation-rbdnd/blob/6774a52768c4284b16e9e4a18360420d11361d19/src/index.jsx#L43-L51

A dummy solution would be to add:

```
componentDidMount() {
    if (this.props.snapshot.isDragging) {
      animationId = requestAnimationFrame(this.patchTransform);
    }
  }
```

I tested in my app and it works like charm, I can come up with a PR if you're interested.

**Comment by rokborf** [comment]
Hey @cluk3 ,

I will take a look over the weekend

**Comment by rokborf** [comment]
@cluk3 , that makes sense, please create a PR

**Comment by rokborf** [comment]
Fixed in 2.1.0 version
